### ISSUE TITLE
Delete button missing

### ISSUE THREAD

**Issue by tikky** [perliedman/leaflet-routing-machine#621]
Hi there,

on your webpage I am missing a button, where I should click to delete waypoint from routing?
![delete_missing](https://user-images.githubusercontent.com/546755/103523744-a371fd80-4e7c-11eb-95b4-9a922d6ab376.png)

I have checked on my phone, and another web browsers and the same problem occurs. So seems to be a bug/problem.

**Comment by Ruschio** [comment]
Which browser is affected from this problem?

**Comment by tikky** [comment]
The problem is visible on: Opera, Chrome, Edge - tested on Windows 10
Problem also on iOS @ iPhone11

Works fine only on Firefox

For reference I have tested main page:
http://www.liedman.net/leaflet-routing-machine/

Please find 3 screenshots
![firefox-ok](https://user-images.githubusercontent.com/546755/103634348-3aa38780-4f47-11eb-96e9-098ed1c70745.PNG)
![chrome-problem](https://user-images.githubusercontent.com/546755/103634351-3b3c1e00-4f47-11eb-8c15-51e9ca899c41.PNG)
![EDGE-problem](https://user-images.githubusercontent.com/546755/103634352-3b3c1e00-4f47-11eb-86c7-e1bcaf5d8ad1.PNG)

**Comment by Ruschio** [comment]
Yes I got it in Chrome too but not on Firefox (as you said).
I looked at console logs on browsers and seems that on Chrome there are several forbidden requests errors, that there aren't on Firefox.
![Chrome](https://user-images.githubusercontent.com/44363237/103635558-fe712680-4f48-11eb-874b-a79bcca49ca6.png)

But I also tried to implement Routing Machine on my test website and works good with Chrome, so probably there is some browsers problems on that site and they don't get resources in the correct way.
![Chrome](https://user-images.githubusercontent.com/44363237/103635787-4e4fed80-4f49-11eb-9d00-5ef542e9b166.png)

**Comment by tikky** [comment]
ok, I will check on my local development env

BTW: on webpage with the problem maybe will be better to use OpenStreetMap tiles - this should resolve the issue

**Comment by tikky** [comment]
I have checked that on my local development site and I am still experiencing the same issue (tested on Chrome and Opera):
![image](https://user-images.githubusercontent.com/546755/103710003-319ccf80-4fb4-11eb-9ffe-5902905ba146.png)

Here is a full html test, co you can check if you can observe the same:
```
<!DOCTYPE html>
<html>
<body>

<link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" />
<script src="https://unpkg.com/leaflet@1.7.1/dist/leaflet.js"></script>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet-routing-machine/3.2.12/leaflet-routing-machine.css" integrity="sha512-eD3SR/R7bcJ9YJeaUe7KX8u8naADgalpY/oNJ6AHvp1ODHF3iR8V9W4UgU611SD/jI0GsFbijyDBAzSOg+n+iQ==" crossorigin="anonymous" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet-routing-machine/3.2.12/leaflet-routing-machine.min.js" integrity="sha512-FW2A4pYfHjQKc2ATccIPeCaQpgSQE1pMrEsZqfHNohWKqooGsMYCo3WOJ9ZtZRzikxtMAJft+Kz0Lybli0cbxQ==" crossorigin="anonymous"></script>

<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/perliedman-leaflet-control-geocoder/2.1.0/Control.Geocoder.min.css" integrity="sha512-XK7G+iGsTdf8/PmVD9/ju65AxCIOaUZly1lY8r9jCNrXO5rGzT2lMYCySGEhb/v8XYXybsZUztXvRdGSyxu/3A==" crossorigin="anonymous" />
<script src="https://cdnjs.cloudflare.com/ajax/libs/perliedman-leaflet-control-geocoder/2.1.0/Control.Geocoder.min.js" integrity="sha512-om2lbFibjXmte7C8BK5Pgx2/keKB8Dywi5iRg/7ZPfTFQvGwUlilBflOPRaoj/Nm8UlF9VpzXq2DRI8f2SlBrw==" crossorigin="anonymous"></script>



<h1>Leaflet routing test</h1>

<div style="height: 600px;" id="mapid">
</div>
<script>
var map = L.map('mapid').setView([50.27264, 7.26469], 13);
console.log(map);
L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png').addTo(map);
var control = L.Routing.control({
  waypoints: [
    L.latLng(50.67, 17.926),
    L.latLng(50.033136, 20.207316)
  ],
  router: new L.Routing.osrmv1({
    language: 'en',
    profile: 'car'
  }),
  collapsible: true,

  geocoder: L.Control.Geocoder.nominatim({})
}).addTo(map);
</script>
</body>
</html>
```

**Comment by Ruschio** [comment]
Ok man, I tried to copy paste your code inside my index page and I got your same issue in Chrome.
I opened style editor and the buttons exist, but there aren't showed because you need to add those code at your style:

`* {
  box-sizing: border-box;
}`

On my previous try I didn't see the issue because of it, so I think adding this code helps to display well all buttons.
Try it and write your feedback!

**Comment by tikky** [comment]
Hello, yes, adding this css seems to resolve an issue.
I've checked on Opera/Chrome - and iPhone/iOS
So is it a bug or I've done something wrong? For me it is a bug cause even on main page of leaflet-routing-machine did not work as expected.

Anyway thank you for your help and information.

One thing which still looks not correct is a button when you collapse the window.
Here is a JSfiddle: https://jsfiddle.net/3f24Ln1x/

and this is how it looks - icon is not centered properly:
![image](https://user-images.githubusercontent.com/546755/103761124-76595280-5016-11eb-9cea-9f12f11750c2.png)

**Comment by Ruschio** [comment]
Yes I think that the responsive style can be done better than now, but basically you'll need to apply this style:

`.leaflet-routing-container, .leaflet-routing-error {
    width: 320px;
    background-color: white;
    padding-top: 4px; <-- remove this
    transition: all 0.2s ease;
    box-sizing: border-box;
}

.leaflet-routing-container-hide .leaflet-routing-collapse-btn {
    display: block;
    width: 100%;
    height: 100%;
    background-image: url('routing-icon.png');
    background-repeat: no-repeat;
    background-position: center;
}

.leaflet-routing-collapse-btn {
    display: table;
    margin-right: 10px;
    font-size: 24px;
    color: #ccc;
    font-weight: bold;
    margin-left: auto;
}`

**Comment by tikky** [comment]
Hello,

have checked.
What works for me is to override

**Comment by tikky** [comment]
Hello,

thanx for suggestions.
What works for me is to override two css settings but in different place:

```
<style>
.leaflet-routing-container-hide .leaflet-routing-collapse-btn {
top: 0px;
left: 1px;
}
</style>
```

Now seems to look fine for me:
![image](https://user-images.githubusercontent.com/546755/103767501-0ac8b280-5021-11eb-9326-4de4ea6a5ef6.png)

and working code example:
https://jsfiddle.net/8ywe7o2z/

**Comment by Ruschio** [comment]
This is not good when change the screen size, so you will need to remove from `padding-top: 4px` from the `.leaflet-routing-container, .leaflet-routing-error` class

**Comment by tikky** [comment]
As suggested have deleted `padding-top: 4px` from the` .leaflet-routing-container, .leaflet-routing-error` class
and changed `.leaflet-routing-container-hide .leaflet-routing-collapse-btn` as per above, but the icon is not centered:

Have to add this two lines in bold to make it work for me:
.leaflet-routing-container-hide .leaflet-routing-collapse-btn {
  display: block;
  width: 100%;
  height: 100%;
  background-image: url('routing-icon.png');
  background-repeat: no-repeat;
  background-position: center;
/* Have to add this two lines: */
  **position: relative;
  left: 0px;**
}


Hers is fiddle, if this still not the way as it should be could you please edit the jsfiddle and reply back with correct one?
https://jsfiddle.net/bn4kqj3L/2/

**Comment by Ruschio** [comment]
Instead of `relative`, you should use `static` and remove `left: 0px` from the style

**Comment by tikky** [comment]
Yes, finally with `static `and without `left: 0px `works fine:
https://jsfiddle.net/z5prxjyw/2/

```
.leaflet-routing-container-hide .leaflet-routing-collapse-btn {
  display: block;
  width: 100%;
  height: 100%;
  background-image: url('https://raw.githubusercontent.com/perliedman/leaflet-routing-machine/master/dist/routing-icon.png');
  background-repeat: no-repeat;
  background-position: center;
  position: static;
}
```

So do we need to propose some changes in source files?

**Comment by Ruschio** [comment]
Maybe yes

**Comment by tikky** [comment]
So, `* { box-sizing: border-box; }` should be added to `leaflet-routing-machine.css` file?
Or in different place?
